src/main/java/com/jfrog/ide/idea/utils/Utils.java

```diff
@@ -39,6 +42,12 @@ public static boolean areRootNodesEqual(DependenciesTree lhs, DependenciesTree r
                 StringUtils.equals(lhsGeneralInfo.getPath(), rhsGeneralInfo.getPath());
     }
 
+    public static boolean gavStringEqual(GeneralInfo lhs, GeneralInfo rhs) {
```

Comment:
Can we replace this method with an equals method in the GeneralInfo class?

Reply:
No because these are only 3 out of 7 class variables of the General info. This is a private use case of general info comparison.